### PR TITLE
Package zarray

### PR DESCRIPTION
**Pull request by davidbrochart**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by davidbrochart** [comment]
@SylvainCorlay @JohanMabille do you agree to be maintainers of this package?

File: recipes/zarray/build.sh [hunk 1]
```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
-￼
```
Comment (ocefpaf) [inline recipes/zarray/build.sh]: For some reason this odd character was your file.

Comment (davidbrochart) [reply recipes/zarray/build.sh]: Thanks a lot!

File: recipes/zarray/bld.bat [hunk 2]
```diff
@@ -0,0 +1,8 @@
+cmake -G "NMake Makefiles" -DCMAKE_INSTALL_PREFIX=%LIBRARY_PREFIX% -DCMAKE_INSTALL_LIBDIR=lib %SRC_DIR%
+if errorlevel 1 exit 1
+
+nmake
+if errorlevel 1 exit 1
+
```
Comment (ocefpaf) [inline recipes/zarray/bld.bat:6]: Are there any `make check/test` or `ctest` that we can run here?

Comment (davidbrochart) [reply recipes/zarray/bld.bat:6]: This was mostly copied from other xtensor packages, maybe @JohanMabille knows?

Comment (ocefpaf) [reply recipes/zarray/bld.bat:6]: Merging this, you can enable tests, if there are any, on the feedstock later.

**Pull request merged**